misc/python/stubs/pg8000/dbapi.pyi

```diff
@@ -7,4 +7,7 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-class ProgrammingError(Exception): ...
+class Error(Exception): ...
+class DatabaseError(Error): ...
+class ProgrammingError(Error): ...
+class InterfaceError(Error): ...
```

Comment:
random q: I tried adding `class InterfaceError(Exception): ...` but that didn't make the pycheck error go away - is there a simple answer why?

Reply:
Probably because you were importing from `pg8000.native` instead of `pg8000.dbapi` above?